### DISCUSSION TITLE
Allow subfolders in the attachments directory

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5827] [Feature Requests]
Windows cannot handle more than 5,000 photos (on average) in a single folder, but users want to be able to access these in their file browser. We should discuss revisiting this.

**Requested By:** Lausanne and New Mexico

**Comment by maxpatiiuk** [comment]
https://answers.microsoft.com/en-us/windows/forum/all/how-many-files-can-a-folder-hold/25368d4b-83e3-47b3-b7a3-a0fb8c7286a7

It could be their network folder limitation or slow computer, because windows by itself should support that
